diff --git a/dist/server/lib/start-server.js b/dist/server/lib/start-server.js
index c274e6e205f847fd5e393f819276e1af92b41d63..7c6589e6cd3c888647db077956b000dae5314654 100644
--- a/dist/server/lib/start-server.js
+++ b/dist/server/lib/start-server.js
@@ -39,6 +39,17 @@ const _appinfolog = require("./app-info-log");
 const _turbopackwarning = require("../../lib/turbopack-warning");
 const _trace = require("../../trace");
 const _ispostpone = require("./router-utils/is-postpone");
+const _logger = require("pino-http")({
+  transport: {
+    target: "pino/file",
+    options: {
+      destination: '/tmp/output/output.log',
+      mkdir: true,
+      all: true,
+      translateTime: true,
+    },
+  },
+});
 function _interop_require_default(obj) {
     return obj && obj.__esModule ? obj : {
         default: obj
@@ -132,6 +143,7 @@ async function startServer(serverOptions) {
         throw new Error("Using a self signed certificate is only supported with `next dev`.");
     }
     async function requestListener(req, res) {
+        _logger(req, res);
         try {
             if (handlersPromise) {
                 await handlersPromise;
